test/dlc_tests/sanity/test_pt_s3.py

```diff
@@ -14,12 +14,14 @@
 
 @pytest.mark.integration("pt_s3_plugin_sanity")
 @pytest.mark.model("N/A")
-def test_pt_s3_sanity(pytorch_training, pt16_and_above_only):
+def test_pt_s3_sanity(pytorch_training):
     """
     Check that the internally built PT S3 binary is properly installed.
     :param pytorch_training: framework fixture for pytorch training
     """
     _, framework_version = get_framework_and_version_from_tag(pytorch_training)
+    if Version(framework_version) < Version("1.8"):
+        pytest.skip("S3 plugin sanity is supported on PyTorch version >=1.8")
```

Comment:
Please undo this, and allow this to be fixed in the PR dedicated to this.

Reply:
I have added this as tests fail if its not there. Other PR can remove it after merging mainline.
Without this code, it blocks the PT1.7.1 release if there is some issue